### PR TITLE
feat(icon): Support for Netin System templates schema

### PR DESCRIPTION
**Pull request by imartingon**
**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

Let's add a Netin System [templates](https://docs.netin.io/netin_webUI/NetinDS/templates/#plantillas-templates) icon. Extensions .ndst.yml .ndst.json and ndst.yaml

### CODE REVIEW

**Review by cjimenezsaiz — APPROVED** [review]

**Review by cjimenezsaiz — APPROVED** [review]

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -4552,5 +4552,10 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedExtensions.ts:4555]: Please, use alphabetical order.

File: src/iconsManifest/supportedExtensions.ts [hunk 2]
```diff
@@ -4552,5 +4552,10 @@ export const extensions: IFileCollection = {
...
+      icon: 'ndst',
+      extensions: ['ndst.yaml', 'ndst.yml', 'ndst.json'],
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedExtensions.ts:4557]: Are these extensions or filenames? Just to be sure.

**Review by robertohuertasm — CHANGES_REQUESTED** [review]

File: src/iconsManifest/supportedExtensions.ts (hunk 2 above)
Comment (imartingon) [reply src/iconsManifest/supportedExtensions.ts:4557]: Are extensions.

File: src/iconsManifest/supportedExtensions.ts (hunk 1 above)
Comment (imartingon) [reply src/iconsManifest/supportedExtensions.ts:4555]: Ok i think now its ok. Thanks.

**Pull request merged**